Source/Fuse.Controls.Navigation/NavigationControl.Pages.uno

```diff
@@ -34,8 +34,7 @@ namespace Fuse.Controls
 			
 			if (_pageHistory == null)
 				return;
-			
-			var obs = _pageHistory as IObservable;
+			var obs = _pageHistory as IObservableArray;
```

Comment:
It's not quite clear to me why change is needed. it is done in 01fd5efe68d018b3c2a95771b0cf95a8dd5e3531 while adding a test, which seems a bit odd... Is this a bug-fix? If so, I think we need a better explanation in the commit message...

Reply:
That seems to be a squash of these two commits here: https://github.com/sebbert/fuselibs-public/compare/36742cb3c2070b63dbcf8f62975ba71de5960f75~2...36742cb3c2070b63dbcf8f62975ba71de5960f75

Reply:
Many places that used `IObservable` before only actually need an `IObservableArray` now (which is a base class to `IObservable`. This is part of the type cleanup in the bindings.

Reply:
Then this seems to be squashed into the wrong commit.